test/test_vid.py

```
@@ -39,6 +40,20 @@ def test_generate_frames_cv2_rtsp(rtsp_server: str):
     assert seen > 0
 
 
+def test_generate_latest_frames_cv2_rtsp(rtsp_server: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_generate_latest_frames_cv2_rtsp`

Add return type annotation: `None`

(ANN201)